src/main/java/org/takes/rq/RqHeaders.java

```diff
@@ -93,7 +93,7 @@ public Base(final Request req) {
         public List<String> header(final CharSequence key)
             throws IOException {
             final List<String> values = this.map().get(
-                key.toString().toLowerCase(Locale.ENGLISH)
+                new EnglishLowerCase(key.toString()).string()
```

Comment:
@dalifreire I think it would better to create one more constructor for `EnglishLowerCase` which accepts `CharSequence`